core/src/main/java/org/mule/runtime/core/internal/streaming/object/ManagedCursorIteratorProvider.java

```diff
@@ -26,34 +27,36 @@
   /**
    * {@inheritDoc}
    */
-  public ManagedCursorIteratorProvider(CursorContext cursorContext, CursorManager cursorManager) {
-    super(cursorContext, cursorManager);
+  public ManagedCursorIteratorProvider(CursorContext cursorContext, CursorManager cursorManager,
+                                       MutableStreamingStatistics statistics) {
+    super(cursorContext, cursorManager, statistics);
   }
 
   /**
    * {@inheritDoc}
    */
   @Override
   protected CursorIterator managedCursor(CursorIterator cursor, CursorContext handle) {
-    return new ManagedCursorIterator(cursor, handle);
+    return new ManagedCursorIterator(cursor);
   }
 
   private class ManagedCursorIterator<T> implements CursorIterator<T> {
 
     private final CursorIterator<T> delegate;
-    private final CursorContext cursorContext;
+    //private final CursorContext cursorContext;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7643&issues=AWirZZoxotIXnmsUDp_u&open=AWirZZoxotIXnmsUDp_u)

[ISSUE_KEY]: AWirZZoxotIXnmsUDp_u